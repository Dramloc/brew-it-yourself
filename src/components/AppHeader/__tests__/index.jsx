import render from 'preact-render-spy';
import { AppHeader } from '../index';

describe('<AppHeader />', () => {
	test('<AppHeader /> matches snapshot', () => {
		expect(render(<AppHeader />)).toMatchSnapshot();
	});
});
