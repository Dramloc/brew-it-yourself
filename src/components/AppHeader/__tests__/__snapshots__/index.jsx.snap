// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> <AppHeader /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<header class="AppHeader">
  <div>
    <img
      src="test-file-stub"
      alt="The Brew It Yourself logo: hand drawn hops"
     />
    <h1>Brew It Yourself</h1>
    <nav>
      <a
        onClick={[Function m]}
        href="/"
        class="AppHeader_Link AppHeader_Link__active"
      >
        <span>Recipes</span>
      </a>
    </nav>
  </div>
</header>

`;
